docs/admin/30-configuration/05-environment-variables.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 1
+id: env-variables
+title: Environment Variables
```

Comment:
If you would like to configure something and look it up in the docs, would you google for something like "Opencloud Config Options"? If yes, we should call the navitem label like this, meaning eg. ~Environment Variables~ ->  **Config Options**
Calling it "Environment Variables" assumes, that first time users know, that OpenCloud gets configured via Environment Variables. I doubt that. What do you think @Svanvith @AlexAndBear ?